### PR TITLE
updated preview instructions

### PR DESCRIPTION
**Pull request by kailuowang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,11 @@ You can also use `tut` in posts. See `posts/2016-09-30-subtype-typeclasses.md` f
 
 To preview your changes, you have to install [Rake](https://ruby.github.io/rake/) and [Bundler](https://bundler.io/) first.
 To download and set up all necessary dependencies, run
-
+You might need to run install bundler first
+```bash
+$ gem install bundler
```
Comment (larsrh) [inline README.md]: That's what `rake init` does.

Comment (kailuowang) [reply README.md]: It didn't do the trick for me, complaining i need bundle 2. I was on bundle 1.x

Comment (larsrh) [reply README.md]: Oh, I misread what you wrote. This is a pending issue (#251); I will address it separately.

Comment (kailuowang) [reply README.md]: okay I will remove this for now.

**Pull request merged**